src/test/java/com/microsoft/sqlserver/jdbc/preparedStatement/BatchExecutionWithBulkCopyTest.java

```diff
@@ -570,7 +571,7 @@ public void testIllegalNumberOfArgNoColumnList() throws Exception {
 
             pstmt.executeBatch();
             fail(TestResource.getResource("R_expectedExceptionNotThrown"));
-        } catch (BatchUpdateException e) {
+        } catch (BatchUpdateException | SQLServerException e) {
```

Comment:
Can you explain why you have to do this?

Reply:
fails thru a different code path for AE